comms/src/peer_manager/migrations/v3.rs

```diff
@@ -0,0 +1,115 @@
+//  Copyright 2020, The Tari Project
+//
+//  Redistribution and use in source and binary forms, with or without modification, are permitted provided that the
+//  following conditions are met:
+//
+//  1. Redistributions of source code must retain the above copyright notice, this list of conditions and the following
+//  disclaimer.
+//
+//  2. Redistributions in binary form must reproduce the above copyright notice, this list of conditions and the
+//  following disclaimer in the documentation and/or other materials provided with the distribution.
+//
+//  3. Neither the name of the copyright holder nor the names of its contributors may be used to endorse or promote
+//  products derived from this software without specific prior written permission.
+//
+//  THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES,
+//  INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+//  DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+//  SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+//  SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY,
+//  WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE
+//  USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+use crate::{
+    net_address::MultiaddressesWithStats,
+    peer_manager::{
+        connection_stats::PeerConnectionStats,
+        migrations::Migration,
+        node_id::deserialize_node_id_from_hex,
+        NodeId,
+        Peer,
+        PeerFeatures,
+        PeerFlags,
+        PeerId,
+    },
+    protocol::ProtocolId,
+    types::CommsPublicKey,
+};
+use chrono::NaiveDateTime;
+use log::*;
+use serde::{Deserialize, Serialize};
+use std::collections::HashMap;
+use tari_crypto::tari_utilities::hex::serialize_to_hex;
+use tari_storage::{
+    lmdb_store::{LMDBDatabase, LMDBError},
+    IterationResult,
+};
+
+const LOG_TARGET: &str = "comms::peer_manager::migrations::v2";
```

Comment:
I know this is merged now, but just wanted to check if this should be `::v3` instead?

Reply:
Yes This should be, thanks for pointing out. Fixing in my current code

Reply:
Nice one 👍

Reply:
#2318